bindings/python/iota_sdk/types/input.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+from dataclasses import dataclass
+from enum import IntEnum
+from iota_sdk.types.common import HexStr
+
+
+class InputType(IntEnum):
+    """Input types.
+
+    Attributes:
+        Utxo: An unspent transaction output.
+        Treasury: The treasury output.
+    """
+    Utxo = 0
+    Treasury = 1
+
+
+@dataclass
+class UtxoInput:
+    """Represents an input referencing an output.
+
+    Attributes:
+        type: The type of input.
+        transactionId: The transaction id that created the output.
+        transactionOutputIndex: The output index.
+    """
+    type: int
+    transactionId: HexStr
+    transactionOutputIndex: int
```

Comment:
Is that really the serialized name?

Reply:
Yes https://api.testnet.shimmer.network/api/core/v2/blocks/0xc27f1bd5dbd719ca19d98a351eca6ff2e32bbd6791351f2475cc9fdff02d8323

Reply:
Weird, ok